packages/sanity/src/core/form/inputs/PortableText/object/BlockObjectActionsMenu.tsx

```diff
@@ -24,7 +24,6 @@ import {IntentLink} from 'sanity/router'
 
 interface BlockObjectActionsMenuProps extends PropsWithChildren {
   focused: boolean
-  isActive?: boolean
```

Comment:
This was incorrectly sent as a always true variable (line 320 above).

Remove it and use prop `focused` instead.